lib/main_common.pm

```diff
@@ -2796,6 +2796,12 @@ sub load_publiccloud_tests {
         loadtest "publiccloud/img_proof";
     }
     elsif (get_var('PUBLIC_CLOUD_QAM')) {
+        if (get_var('PUBLIC_CLOUD_QAM_IMGPROOF') ne "") {
+            my $tunneled = get_var("TUNNELED");
+            set_var("TUNNELED", "");
+            loadtest "publiccloud/img_proof";
```

Comment:
This "loadtest" function just put the module in a array. But id doesn't execute anything.
So if the `run()` method is called from `publiccloud/img_proof` the `TUNNELED` variable is still set.

Isn't it that we have in that `TUNNELED` thingy already a instance up and running. So maybe we can just omit these two lines https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/publiccloud/img_proof.pm#L54 and reuse yours?

On the other hand, `img-proof` is rebooting the instance so, that it also get a new IP address. Which would then cause a resetup of all our `TUNNELED` consoles. So from current point of view, it is more easy to launch a second instance.

Reply:
Thanks for the input, I'm playing around with this right now, trying to find a nice way to get this running

Reply:
The problem is, that we need `TUNNELED=1` for the other tests in our runs and I was trying to avoid modifications on `img_proof.pm`.

Reply:
Is our virtual console also used for `TUNNELED` access to SUT, I guess so?
We could use `VIRTIO_CONSOLE_NUM=2` and select the second virtual one with `select_console('root-virtio-terminal1')` in img-proof if `TUNNELED` is set. 
It's hacky, but the smallest impact.

Or you just run this standalone, meaning not within this job. We do this in "other" publiccloud tests.

Reply:
My solution was in the end to create a `enable_tunnel` and a `disable_tunnel` test (which do nothing else than set `TUNNELED` to `1` or `0` respectively). That seems to work in my current verification runs

Reply:
cool, can I take a look to that VR?

Reply:
I guess it the one from the first comment :)

I think I was just mistaken, the easy fact to run ipa and other testmodules before `ssh_interactive_start`? This actually already solve it, or do you really need this `TUNNELED` set/unset thing?

Reply:
Hi Clemens, much appreciated input here :-)

I was playing around with the `TUNNELED` setting to avoid touching `img_proof.pm`, but it appears this is anyways needed.
We pass the instance and provider as argument to our tests and I needed to update `img_proof.pm` accordingly.


Anyway - this is still very much work in progress!

Reply:
Hi Felix, you are welcome! And for sure this is "very much" WIP, I'm aware of it.

> I was playing around with the `TUNNELED` setting to avoid touching `img_proof.pm`, but it appears this is anyways needed.
> We pass the instance and provider as argument to our tests and I needed to update `img_proof.pm` accordingly.
I'm not sure if we are aligned here.
From my perspective (without doing any tests just looking at your VRs) it is not needed, if you run such tests before you start with `ssh_interactive_start`.
Reason:
* in `img_proof.pm` we do launch a instance and delete it afterwards. 
* in "your" tests, you run a instance and pass it from one test-module to the other
* in "your" tests, you are using this TUNNELED feature to have a transparent usage of different consoles but these consoles are actually connected to the PC instance via SSH

=> if you you now using the same instance to run img-proof on it, the instances get rebooted and will also get new external IP address. I'm not sure if our code is aware to catch that cases.

Reply:
Thanks for pointing this out. I think I am aware of this issue and we account for it, but I would very much invite you to look critical at the PR in this matter.

The good news is, that we found a way to run this test with `TUNNELED=0`. Still playing around with it, but it looks promising and much more aligned to a common way here.

Reply:
We figured a way out to make the `img_proof` test run work with `TUNNELED=0`. Please feel free to review our PR :slightly_smiling_face: